Explorer/Assets/DCL/SDKComponents/Tween/Systems/TweenLoaderSystem.cs

```diff
@@ -1,28 +1,20 @@
 using Arch.Core;
 using Arch.System;
 using Arch.SystemGroups;
-using Arch.SystemGroups.Throttling;
 using DCL.Diagnostics;
 using DCL.ECSComponents;
 using DCL.SDKComponents.Tween.Components;
-using DCL.SDKComponents.Tween.Helpers;
 using ECS.Abstract;
-using ECS.Unity.Groups;
-using UnityEngine;
-using UnityEngine.Pool;
+using ECS.Groups;
 
 namespace DCL.SDKComponents.Tween.Systems
 {
-    [UpdateInGroup(typeof(ComponentInstantiationGroup))]
+    [UpdateInGroup(typeof(SyncedSimulationSystemGroup))]
     [LogCategory(ReportCategory.TWEEN)]
-    [ThrottlingEnabled]
```

Comment:
I recommend you to split the system into 2: As for `LoadTween` you should throttle but for `UpdateTween` you should not

Reply:
Agreed and done!